adapters/tlsio_openssl.c

```diff
@@ -390,7 +390,8 @@ static const IO_INTERFACE_DESCRIPTION tlsio_openssl_interface_description =
 };
 
 static LOCK_HANDLE * openssl_locks = NULL;
-
+static int locks_init_count = 0;
```

Comment:
>locks_init_count [](start = 11, length = 16)

Nitpick: Consider renaming to locks_install_count

Reply:
Fixed.